<!DOCTYPE html>

<html>
  <head>
     <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());
      gtag('config', 'G-JS8TMD6Z1C');
    </script>

    <title>Brian Hie | Publications</title>
    <meta http-equiv="Content-Type" content="text/html;charset=ISO-8859-1"> 
    <meta name="description" content="Brian Hie's personal website">
    <meta name="keywords" content="brian,hie,stanford,mit,csail,san,diego">
    <meta name="author" content="Brian Hie">
    <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:400|Barlow:400italic,400,300italic,300"
          rel="stylesheet" type="text/css">
    <link rel="stylesheet" type="text/css" href="brianhie.css">
  </head>

  <body>

    <div class="container">

      <a href="http://brianhie.com">
      <div class="home_menu">
        <p class="home_title title_font home_image">
          Brian Hie
        </p>
      </div>
      </a>

      <div class="about sans_font">
        <p class="blog_header" style="margin-bottom: 2px;">Preprints</p>
        <p class="blog_body" style="font-size: 14px;">
          &#42;Equal contribution. <sup>&#8224;</sup>Corresponding author.
        </p>
        
        <p class="blog_body">
          <b>B. Hie</b>&#42;, S. Candido&#42;, Z. Lin, O. Kabeli, R. Rao, N. Smetanin, T. Sercu, and A. Rives. <br/>
          <b><a target="_newtab" href="https://www.biorxiv.org/content/10.1101/2022.12.21.521526v1">&ldquo;A high-level programming language for generative protein design.&rdquo;</a></b> <br/>
          <i>bioRxiv</i>, DOI:10.1101/2022.12.21.521526 (2022). <b><a target="_newtab" href="https://github.com/facebookresearch/esm/tree/main/examples/protein-programming-language">[Code]</a></b>
        </p>
        
        <div class="about sans_font">
        <p class="blog_header" style="margin-bottom: 2px;">Publications</p>
        <p class="blog_body" style="font-size: 14px;">
          &#42;Equal contribution. <sup>&#8224;</sup>Corresponding author.
        </p>

        <p class="blog_body">
          <b>B. Hie</b><sup>&#8224;</sup>, V. Shanker, D. Xu, T.U.J. Bruun, P.A. Weidenbacher, S. Tang, W. Wu, J.E. Pak, and P.S. Kim<sup>&#8224;</sup>. <br/>
          <b><a target="_newtab" href="https://www.nature.com/articles/s41587-023-01763-2">&ldquo;Efficient evolution of human antibodies from general protein language models.&rdquo;</a></b> <br/>
          <i>Nature Biotechnology</i>, DOI:10.1038/s41587-023-01763-2 (2023). <b><a target="_newtab" href="https://github.com/brianhie/efficient-evolution">[Code]</a></b>
        </p>
          
        <p class="blog_body">
          Z. Lin&#42;, H. Akin&#42;, R. Rao&#42;, <b>B. Hie</b><sup>&#42;</sup>, Z. Zhu, W. Lu, N. Smetanin, R. Verkuil, O. Kabeli, Y. Shmueli, A. dos Santos Costa, M. Fazel-Zarandi, T. Sercu, S. Candido, and A. Rives. <br/>
          <b><a target="_newtab" href="https://www.science.org/doi/10.1126/science.ade2574">&ldquo;Evolutionary-scale prediction of atomic level protein structure with a language model.&rdquo;</a></b> <br/>
          <i>Science</i>, 379:6637 (2023). <b><a target="_newtab" href="https://github.com/facebookresearch/esm">[Code]</a></b>
        </p>
        
        <p class="blog_body">
          <b>B. Hie</b><sup>&#8224;</sup>, K.K. Yang, and P.S. Kim<sup>&#8224;</sup>. <br/>
          <b><a target="_newtab" href="https://www.sciencedirect.com/science/article/pii/S2405471222000382">&ldquo;Evolutionary velocity with protein language models predicts evolutionary dynamics of diverse proteins.&rdquo;</a></b> <br/>
          <i>Cell Systems</i>, 13:4 (2022). <b><a target="_newtab" href="https://github.com/brianhie/evolocity">[Code]</a></b>
        </p>

        <p class="blog_body">
          M.C. Maher, I. Bartha, S. Weaver, J. di Iulio, E. Ferri, L. Soriaga, F.A. Lempp, <b>B. Hie</b>, B. Bryson, B. Berger, D.L. Robertson, G. Snell, D. Corti, H.W. Virgin, S.L. Kosakovsky Pond, and A. Telenti. <br/>
          <b><a target="_newtab" href="https://www.science.org/doi/10.1126/scitranslmed.abk3445">&ldquo;Predicting the mutational drivers of future SARS-CoV-2 variants of concern.&rdquo;</a></b> <br/>
          <i>Science Translational Medicine</i>, DOI: 10.1126/scitranslmed.abk3445 (2022).
        </p>
        
        <p class="blog_body">
          <b>B. Hie</b> and K. Yang. <br/>
          <b><a target="_newtab" href="https://www.sciencedirect.com/science/article/pii/S0959440X21001457">&ldquo;Adaptive machine learning for protein engineering.&rdquo;</a></b> <br/>
          <i>Current Opinion in Structural Biology</i>, 72:145-152 (2022).
        </p>
        
        <p class="blog_body">
          <b>B. Hie</b>.<br/>
          <b><a target="_newtab" href="https://dspace.mit.edu/handle/1721.1/139231">&ldquo;Algorithms for understanding and fighting infectious disease.&rdquo;</a></b> <br/>
          Massachusetts Institute of Technology, Doctoral Thesis (2021).
        </p>
        
        <p class="blog_body">
          R. Singh&#42;, <b>B. Hie&#42;</b>, A. Narayan, and B. Berger.<br/>
          <b><a target="_newtab" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02313-2">&ldquo;Schema: Metric learning enables interpretable synthesis of heterogeneous single-cell modalities.&rdquo;</a></b> <br/>
          <i>Genome Biology</i>, 22:131 (2021). <b><a target="_newtab" href="https://github.com/rs239/schema">[Code]</a></b>
        </p>

        <p class="blog_body">
          <b>B. Hie</b>, E. Zhong, B. Berger, and B. Bryson. <br/>
          <b><a target="_newtab" href="https://science.sciencemag.org/content/371/6526/284">&ldquo;Learning the language of viral evolution and escape.&rdquo;</a></b> <br/>
          <i>Science</i>, 371:6526 (2021). A previous version appeared at <b><a target="_newtab" href="https://papers.nips.cc/paper/2020/hash/6754e06e46dfa419d5afe3c9781cecad-Abstract.html">NeurIPS 2020</a></b>. <b><a target="_newtab" href="https://github.com/brianhie/viral-mutation">[Code]</a></b>
        </p>
        
        <p class="blog_body">
          <b>B. Hie</b>, B. Bryson, and B. Berger. <br/>
          <b><a target="_newtab" href="https://www.cell.com/cell-systems/fulltext/S2405-4712(20)30364-1">&ldquo;Leveraging uncertainty in machine learning accelerates biological discovery and design.&rdquo;</a></b> <br/>
          <i>Cell Systems</i>, 11:5 (2020). <b><a target="_newtab" href="https://github.com/brianhie/uncertainty">[Code]</a></b>
        </p>
        
        <p class="blog_body">
          <b>B. Hie&#42;</b>, J. Peters&#42;, S. Nyquist&#42;, A. Shalek, B. Berger, and B. Bryson.<br/>
          <b><a target="_newtab" href="http://www.annualreviews.org/eprint/RRDTIVPBCURYJUEYKYDR/full/10.1146/annurev-biodatasci-012220-100601">&ldquo;Computational methods for single-cell RNA sequencing.&rdquo;</a></b> <br/>
          <i>Annual Review of Biomedical Data Science</i>, 3:1 (2020).
        </p>
        
        <p class="blog_body">
          <b>B. Hie</b>&#42;, H. Cho&#42;, B. DeMeo, B. Bryson, and B. Berger.<br/>
          <b><a target="_newtab" href="https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30152-8">&ldquo;Geometric sketching compactly summarizes the single-cell transcriptomic landscape.&rdquo;</a></b> <br/>
          <i>Cell Systems</i>, 8:6 (2019). A previous version appeared at <b><a target="_newtab" href="https://recomb2019.org/materials/">RECOMB 2019</a></b>. <b><a target="_newtab" href="https://github.com/brianhie/geosketch">[Code]</a></b>
        </p>
        
        <p class="blog_body">
          <b>B. Hie</b>, B. Bryson, and B. Berger.<br/>
          <b><a target="_newtab" href="https://www.nature.com/articles/s41587-019-0113-3"> &ldquo;Efficient integration of heterogeneous single-cell transcriptomes using Scanorama.&rdquo;</a></b> <br/>
          <i>Nature Biotechnology</i>, 37:685-691 (2019). <b><a target="_newtab" href="https://github.com/brianhie/scanorama">[Code]</a></b>
        </p>
                
        <p class="blog_body">
          A.K. Tehranchi, <b>B. Hie</b>, M. Dacre, I.M. Kaplow, K.P. Pettie, P.A. Combs, and H.B. Fraser. <br/>
          <b><a target="_newtab" href="https://elifesciences.org/articles/39595">&ldquo;Fine-mapping cis-regulatory variants in diverse human populations.&rdquo;</a></b> <br/>
          <i>eLife</i>, 8:e39595 (2019).
        </p>
        
        <p class="blog_body">
          <b>B. Hie</b>&#42;, H. Cho&#42;, and B. Berger.<br/>
          <b><a target="_newtab" href="http://science.sciencemag.org/content/362/6412/347">&ldquo;Realizing private and practical pharmacological collaboration.&rdquo;</a></b> <br/>
          <i>Science</i>, 362:6417 (2018). <b><a target="_newtab" href="https://github.com/brianhie/secure-dti">[Code]</a></b>
        </p>

        <p class="blog_body">
          A.K. Tehranchi, M. Myrthil, T. Martin, <b>B. Hie</b>, D. Golan, and H.B. Fraser. <br/>
          <b><a target="_newtab" href="http://www.cell.com/cell/abstract/S0092-8674(16)30339-7">&ldquo;Pooled ChIP-seq links variation in transcription factor binding to complex disease risk.&rdquo;</a></b> <br/>
          <i>Cell</i>, 165:3 (2016).
        </p>
      </div>
    </div>

</body>

</html>
